x/src/test.rs

```diff
@@ -17,16 +23,66 @@ pub struct Args {
     #[structopt(long, short)]
     /// Only run unit tests
     unit: bool,
+
+    #[structopt(long)]
+    /// Do not fast fail the run if tests (or test executables) fail.
+    no_fail_fast: bool,
```

Comment:
mmm, we have `no_fail_fast` here and `fail_fast` in the lint engine. I think the defaults for both make sense but at some point we may want to have a `fail_mode` or similar.

Reply:
gotta love consistency.   I could see fail_mode being needed at some point, but I doubt that x fmt will ever care as I would assume it would run all crates together and not have internal subcommands that mail fail.   Sill good point....